index.bs

```diff
@@ -5595,8 +5595,7 @@ and the return type and argument list (matching <emu-nt><a href="#prod-Type">Typ
 and <emu-nt><a href="#prod-ArgumentList">ArgumentList</a></emu-nt>) on the right side of the equals
 sign gives the signature of the callback function type.
 
-[=Callback functions=] must not
-be used as the type of a [=constant=].
+[=Callback functions=] must not be used as the type of a [=constant=].
```

Comment:
This duplicates the statement on line 6367. I think this should be removed, as the sentence is better fit in the types section.

Reply:
Wait never mind, I misunderstood the PR.